src/index.html

```diff
@@ -4,9 +4,378 @@
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Catalog</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com"
+    crossorigin="anonymous">
+    <link href="https://fonts.googleapis.com/css2?family=Montserrat:wght@700&family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <a href="#" class="header__home-link">
+        <img
+          src="./images/logo.png"
+          alt="Moyo logo"
+          class="header__logo"
+        >
+      </a>
+
+      <nav class="navigation">
+        <menu class="navigation__list">
+          <li class="navigation__item">
+            <a class="navigation__link is-active" href="#apple">Apple</a>
```

Comment:
```suggestion
            <a class="navigation__link navigation__link--is-active" href="#apple">Apple</a>
```

Reply:
But accirdung ti the instruction we should use a class, not a modificator:  "add class `is-active` to the first link (`Apple`) in navigation"

Reply:
> But according ti the instruction we should use a class, not a modificator: "add class `is-active` to the first link (`Apple`) in navigation"

Ok, it's strange but true.